### DISCUSSION TITLE
Cloud account sync not working?

### DISCUSSION THREAD

**Discussion by jrausch12** [posit-dev/positron#3875] [Q&A]
I'm trying to turn on Cloud Changes so I can sync my settings; however, that seems to be disabled? I click it and nothing happens.

**Comment by juliasilge** [comment]
Do you mean "Settings Sync" like for VS Code, i.e. <https://code.visualstudio.com/docs/editor/settings-sync>? Or something else? Can you share a screenshot about where you are changing a setting?

**Reply by SETOsfk** [reply]
<img width="300" alt="Screenshot 2024-07-06 at 10 14 48" src="https://github.com/posit-dev/positron/assets/110775241/cd8564c2-2fc1-4287-bdef-55c8f21a8e3c">
I think he meant this.

**Reply by ntluong95** [reply]
Yes I think this function doesn't work

**Reply by jrausch12** [reply]
What @SETOsfk posted is what I am referring to.

**Reply by juliasilge** [reply]
Ah, this feature provided by VS Code uses Microsoft servers/telemetry/etc. Our builds of Positron are more like VS Codium than the Microsoft-provided builds of VS Code in that we have turned off all access to such servers and telemetry; see https://github.com/posit-dev/positron/pull/2882.

Specifically for syncing settings, there are some open source alternatives. I have not tried them out myself, but you might [look at this open source option](https://open-vsx.org/extension/Shan/code-settings-sync) or choose another way that VS Codium users like to handle syncing their settings across devices.

**Reply by Robinlovelace** [reply]
This is another option, that seems to be a bit more recently maintained (the one Julia linked to was last updated 6 years ago as far as I can tell): https://github.com/zokugun/vscode-sync-settings

**Reply by Robinlovelace** [reply]
Another update:  https://github.com/zokugun/vscode-sync-settings works! It's bit fiddly to set-up but gives you total control over how you store your sync settings.

![image](https://github.com/user-attachments/assets/6890c171-d7d9-42c9-9368-4e8f09e0531e)

**Reply by muehlbauer-m** [reply]
I also just set up [https://github.com/zokugun/vscode-sync-settings](https://github.com/zokugun/vscode-sync-settings) and it (mostly) works. The last slightly annoying bit that does not, is syncing UI state, even though I activated that option in the settings.yml. Is this because positron diverging functionalites to VSC here? Does anyone know a fix to this?
